tools/quickstart/quickstart/cli.py

```diff
@@ -28,8 +29,24 @@
     type=click.Path(file_okay=False),
     default="trustlines",
 )
+@click.option(
+    "--bridge-config",
+    help="file with bridge configuration",
+    type=click.Path(exists=True, dir_okay=False),
+    required=True,
+)
+@click.option(
+    "--netstats-url", help="URL to netstats server", default="laika", required=True
+)
 @click.option("--project-name", help="project name", default="trustlines")
```

Comment:
I know this is not really part of this PR, but for me it is interesting because of the previous new option. So is the default `trustlines` for the project name just a general term, or does it refer to the `tlbc` network? If so I would argue that the defaults of the options should relate to the same network. But this isn't important at all.

Reply:
This is the name that is used mainly for the docker container namespacing(<projectname>_<servicename>_<number>) so for example (trustlines_bridge-client_1)
This will be something like laika and tlbc later.